src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3463,6 +3463,7 @@ static char* map_or_reserve_memory_aligned(size_t size, size_t alignment, int fi
   }
 
   assert(aligned_base != nullptr, "Did not manage to re-map after %d attempts?", max_attempts);
```

Comment:
Shouldn't we remove previous assertion then?

Reply:
Yes, my mistake.
Will  do this.
Thanks for looking into it!